test/Controller/ProfileControllerTest.php

```diff
@@ -35,6 +35,157 @@ public function it_displays_a_profile_page()
         $this->assertEmpty($crawler->filter('.content-header-profile__details'));
     }
 
+    /**
+     * @test
+     */
+    public function it_displays_a_profile_page_with_public_information()
+    {
+        $client = static::createClient();
+
+        $this->mockApiResponse(
+            new Request(
+                'GET',
+                'http://api.elifesciences.org/profiles/1',
+                ['Accept' => 'application/vnd.elife.profile+json; version=1']
+            ),
+            new Response(
+                200,
+                ['Content-Type' => 'application/vnd.elife.profile+json; version=1'],
+                json_encode([
+                    'id' => '1',
+                    'name' => [
+                        'preferred' => 'Preferred Name',
+                        'index' => 'Index Name',
+                    ],
+                    'emailAddresses' => [
```

Comment:
if the algorithm involves "pick the first one" it would be useful to put a `restricted` as the first one to see it excluded, but this is a obscure corner case

Reply:
Done.